src/fmu/sumo/uploader/_sumofile.py

```diff
@@ -130,6 +130,15 @@ def upload_to_sumo(self, sumo_parent_id, sumoclient, sumo_mode):
             and self.metadata.get("data").get("format") in ["openvds", "segy"]
         ):
             self.metadata["data"]["format"] = "openvds"
+            if "vertical_domain" not in self.metadata["data"]:
+                result.update(
+                    {
+                        "status": "rejected",
+                        "metadata_upload_response_status_code": 500,
+                        "metadata_upload_response_text": "File upload cannot be attempted; seismic data does not have a value for data.vertical_domain."
```

Comment:
Suggest "...missing value for data.vertical_domain"

Reply:
Updated.